libraries/AP_LandingGear/AP_LandingGear.h

```diff
@@ -41,8 +41,18 @@ class AP_LandingGear {
     /// returns true if the landing gear is deployed
     bool deployed();
     
-    enum LG_WOW_State {
+    enum LG_LandingGear_State {
```

Comment:
LG_LandingGear_State is quite a bad name as it translates to Landing Gear Landing Gear state. If the method is just called `get_state` there is no reason why this can't be just LG_State.

Reply:
fixed